src/ring_theory/polynomial.lean

```diff
@@ -225,8 +225,8 @@ is_noetherian_submodule_left.1 (is_noetherian_of_fg_of_noetherian _
 
 end ideal
 
-/-- Hilbert basis theorem. -/
-theorem is_noetherian_ring_polynomial [is_noetherian_ring R] : is_noetherian_ring (polynomial R) :=
+/-- Hilbert basis theorem: a polynomial ring over a noetherian ring is a noetherian ring. -/
+theorem polynomial.is_noetherian_ring [is_noetherian_ring R] : is_noetherian_ring (polynomial R) :=
```

Comment:
Shouldn't this be protected, to avoid conflicts with the root definition `is_noetherian_ring`?

Reply:
```suggestion
protected theorem polynomial.is_noetherian_ring [is_noetherian_ring R] : is_noetherian_ring (polynomial R) :=
```